### PR TITLE
Add bootstrap styling to project

### PR DESCRIPTION
**Pull request by saadyousafarbi**
**WIP**

* Add support for bootstrap styling (http://getbootstrap.com)
* Add basic CSS to logistration forms
* Add styling on Profile view/edit page.

### CODE REVIEW

File: autoshop/core/forms.py [hunk 1]
```diff
@@ -26,11 +26,11 @@ class Meta:
...
         fields = ['first_name', 'last_name', 'username', 'email', 'password']
         help_texts = {
-            'first_name': 'Enter first name:',
-            'last_name': 'Enter last name:',
-            'username': 'Enter username:',
-            'email': 'Enter email:',
-            'password': 'Enter password:',
+            'first_name': 'Enter first name',
+            'last_name': 'Enter last name',
+            'username': 'Enter username',
```
Comment (zubair-arbi) [inline autoshop/core/forms.py]: We need to also tell users that it cannot be change later.
Check `edX` register for example: https://courses.edx.org/register

File: autoshop/core/templates/base.html [hunk 2]
```diff
@@ -1,11 +1,6 @@
...
-</head>
-<body>
-        <div class="jumbotron" style="box-shadow: 2px 2px 4px #000000;">
-            <h2 class="text-center"> Auto-Shop</h2>
-            <h3 class="text-center">A website with information about automobiles for buying/selling purposes.</h3>
-        </div>
-        <a href="login">login</a>
-        <a href="register">register</a>
-</body>
+{% extends "basic.html" %}
```
Comment (zubair-arbi) [inline autoshop/core/templates/base.html]: rename this `basic.html` to `base.html`

File: autoshop/core/templates/basic.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>My Project</title>
+            <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css"
```
Comment (zubair-arbi) [inline autoshop/core/templates/basic.html]: We should add this file as project static

File: autoshop/core/templates/register.html [hunk 4]
```diff
@@ -1,18 +1,12 @@
...
-        </div>
-        <form method="post" >
+{% extends "basic.html" %}
+{% load staticfiles %}
+
+{% block content %}
+        <form method="post">
             {% csrf_token %}
                 <table>
             {{form.as_table}}
```
Comment (zubair-arbi) [inline autoshop/core/templates/register.html]: indentation

File: autoshop/core/templates/register.html [hunk 5]
```diff
@@ -1,18 +1,12 @@
...
-        <title>Auto-shop | Register</title>
-    </head>
-    <body>
-        <div class="jumbotron" style="box-shadow: 2px 2px 4px #000000;">
-            <h2 class="text-center"> Auto-Shop</h2>
-            <h3 class="text-center">A website with information about automobiles for buying/selling purposes.</h3>
-        </div>
-        <form method="post" >
+{% extends "basic.html" %}
+{% load staticfiles %}
```
Comment (zubair-arbi) [inline autoshop/core/templates/register.html]: why we are using this template tag?

**Review by zubair-arbi — CHANGES_REQUESTED** [review]
Please fix these suggestions

File: autoshop/core/static/css/core.css [hunk 6]
```diff
@@ -0,0 +1,31 @@
+span:hover {
+    color: red;
+    position: relative;
```
Comment (zubair-arbi) [inline autoshop/core/static/css/core.css]: We should be getting this file from project static folder

**Review by zubair-arbi — APPROVED** [review]
LGTM 👍 . Rebase, squash and add comment.

**Comment by zubair-arbi** [comment]
Also please add screenshots of logistration pages and profile edit page in the PR description.

**Comment by saadyousafarbi** [comment]
Rebased from commit a04464667e1cd8bb22251c93c3d56532943b62d9

**Pull request merged**